### ISSUE TITLE
`keyboard-navigation`: error with j/k when no files have been selected

### ISSUE THREAD

**Issue by tstirrat** [refined-github/refined-github#8022]
### Description

Receiving this error and keyboard nav does not work (unless you manually click on a file path).

```
ElementNotFoundError: Expected element not found: :target
    at expectElement (chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/assets/npm/select-dom.js:21:11)
    at HTMLBodyElement.runShortcuts (chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/assets/features/keyboard-navigation.js:20:25)
```

### How to replicate the issue + URL

1. Go to https://github.com/tstirrat/mGB/pull/6/files
2. Press `j`/`k` without clicking on any file header

**Expected**
The first file is selected

**Actual**
Nothing is selected and the error shows in the console.log

**Workaround**
Once you click on a file path, which causes the `:target` selector to be found, then KB nav works as expected.

### Extension version

24.10.28

### Browser(s) used

Arc, Chrome

### Token ⚠️

- [X] I set a token in the options page. I understand that some features require a token to work.

**Comment by fregante** [comment]
Broken by https://github.com/refined-github/refined-github/pull/7966

It looked like it was an "expected element" but the code silently worked because current position was `-1` and J pushed it to 0

The fix is to use $optional() for that call.
